test/setup/metrics-server/metrics-server-deployment.yaml

```diff
@@ -23,10 +23,15 @@ spec:
         k8s-app: metrics-server
     spec:
       serviceAccountName: metrics-server
+      volumes:
+      # mount in tmp so we can safely use from-scratch images and/or read-only containers
```

Comment:
should we do this in the 0.2.1 version as well?

Reply:
This wasn't supported till 0.3 https://github.com/kubernetes-incubator/metrics-server/commit/a823af80d438d642c29e038ca5336004b2a8b97e